app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/newdevicenotice/NewDeviceNoticeEmailAccessScreen.kt

```diff
@@ -118,15 +110,17 @@ private fun NewDeviceNoticeEmailAccessContent(
 
 @Suppress("MaxLineLength")
 @Composable
-private fun HeaderContent() {
+private fun HeaderContent(
+    modifier: Modifier = Modifier,
+) {
     Image(
         painter = rememberVectorPainter(id = R.drawable.warning),
         contentDescription = null,
-        modifier = Modifier.size(120.dp),
+        modifier = modifier.size(120.dp),
```

Comment:
The `Modifier` should only ever be applied to the top level component, if you wanted to pass one in like this, you would want to wrap it in a column.